### PR TITLE
Explaining usage of HTTP_PROXY

### PR DESCRIPTION
**Pull request by luebken**

### CODE REVIEW

**Comment by luebken** [comment]
ping @kordless

File: content/reference/installation.md [hunk 1]
```diff
@@ -58,6 +58,16 @@ We recommend to make the `swarm` binary available in your PATH by copying it to
 $ sudo cp swarm /usr/local/bin/
 ```
 
+## Using a HTTP proxy
```
Comment (marians) [inline content/reference/installation.md]: I would say it's "... an HTTP proxy"

File: content/reference/installation.md [hunk 2]
```diff
@@ -58,6 +58,16 @@ We recommend to make the `swarm` binary available in your PATH by copying it to
...
+
+The swarm client you install is programmed in [Go](http://golang.org/). That way we can leverage several build in features. For example you can configure a proxy for the connection between the CLI and the swarm cluster by simply setting the HTTP_PROXY env variable.
+
+In Bash this would be:
+```
```
Comment (marians) [inline content/reference/installation.md]: Please add `nohighlight` to prevent rnadom highlighting

**Comment by marians** [comment]
Commented tiny things.

**Comment by marians** [comment]
LGTM

**Comment by kordless** [comment]
I'm going to update the example provided with the naming convention I used on the API reference page.

**Pull request merged**